samples/star-wars-api/HttpHandlers.fs

```diff
@@ -152,22 +226,25 @@ module HttpHandlers =
                 // let! request = ctx.BindJsonAsync<GQLRequestContent>()
                 let query = request.Query
 
-                logger.LogTrace($"Executing GraphQL query:{Environment.NewLine}{query}", query)
+                logger.LogTrace ($"Executing GraphQL query:{Environment.NewLine}{{query}}", query)
                 let operationName = request.OperationName |> Skippable.toOption
-                operationName |> Option.iter (fun on -> logger.LogTrace($"GraphQL operation name: '{operationName}'", on))
+
+                operationName
+                |> Option.iter (fun on -> logger.LogTrace ($"GraphQL operation name: {{on}}", on))
```

Comment:
What was the idea of shortening a variable name?

Reply:
The change I put in makes it log the Some contents of the Option, rather than the Option itself.  The `on` at the end was already in place when I saw it.

Reply:
I mean this. I agree that it was a mistake
```suggestion
                |> Option.iter (fun on -> logger.LogTrace ($"GraphQL operation name: {{operationName}}", on))
```